tests/robot-tests/tests/admin/bau/upload_files.robot

```diff
@@ -35,23 +35,19 @@ Upload a ZIP file subject
     user clicks button    Upload data files
 
     user waits until h2 is visible    Uploaded data files
-    user waits until page contains accordion section    Absence in PRUs
-    user opens accordion section    Absence in PRUs
-
-    ${section}=    user gets accordion section content element    Absence in PRUs
+    user waits until page contains data uploads table
 
     # To ensure "Data file size" and "Number of rows" will be filled
     user waits until page does not contain    Queued    %{WAIT_MEDIUM}
-
-    user checks headed table body row contains    Subject title    Absence in PRUs    ${section}
-    user checks headed table body row contains    Data file    absence_in_prus.csv    ${section}
-    user checks headed table body row contains    Metadata file    absence_in_prus.meta.csv    ${section}
-    user checks headed table body row contains    Data file size    141 Kb    ${section}
-    user checks headed table body row contains    Number of rows    612    ${section}
-    user checks headed table body row contains    Status    Complete    ${section}    %{WAIT_DATA_FILE_IMPORT}
+    user checks table cell contains    row=1    column=1    expected=Absence in PRUs    parent=testid:Data files table
```

Comment:
Why use named parameters? This is pretty inconsistent with how we typically use keywords. Including the parameter names makes the lines much more noisy and hinders readability.

In generally, we use named parameters sparingly throughout the tests e.g. to set optional parameters. We don't usually use them for required parameters like this.

Would revert back to using positional arguments only. Same applies for rest of Robot tests where we've made this change.

Reply:
I feel it makes the test easier to understand, instead of seemingly random strings appended to the end of user created keywords/functions - it's not immediately obvious what the function is doing in my opinion (which is a robot newb's opinion)

I'll remove the names